R/truncate_interlace.R

```diff
@@ -80,7 +80,9 @@
 truncate_interlace <- function(primary, secondary = NULL) {
 
   # Check edge case
-  if (is.null(secondary)) return(primary)
+  if (is.null(secondary) || (is.list(secondary) && length(secondary) == 0)) {
```

Comment:
Please add parentheses to ease reading
```suggestion
  if (is.null(secondary) || (is.list(secondary) && length(secondary) == 0)) {
```

Reply:
Done!